oct/cli/provision/local/all_in_one.py

```diff
@@ -83,11 +82,10 @@ class Stage(object):
     '--provider',
     '-p',
     type=Choice([
-        Provider.libvirt,
         Provider.virtualbox,
         Provider.vmware,
     ]),
-    default=Provider.libvirt,
+    default=Provider.virtualbox,
```

Comment:
The entire local provisioning is non-functional at this point, so I think we could even remove the whole command tree.